metadata-protobuf/proto/Storage.proto

```diff
@@ -46,3 +48,31 @@ message DistributionBucketFamilyMetadata {
   repeated GeographicalArea areas = 3; // Standarized geographical areas covered by the family (providing [{}] will unset the current value)
   repeated string latency_test_targets = 4; // List of targets (hosts/ips) best suited latency measurements for this family
 }
+
+message NodeOperationalStatusMetadata {
+  enum OperationalStatus {
+    // Node is operating normally
+    NORMAL = 0;
+
+    // Node is not operational 
+    NO_SERVICE = 1;
+  }
+
+  // Node's Operational status to set
+  optional OperationalStatus status = 1; 
+
+  // Date (ISO format) from which the node won't be operational (Should be set if status is NoServiceFrom or NoServiceDuring)
+  optional string no_service_from = 2; 
+
+  // Date (ISO format) until which the node won't be operational (Should be set if status is NoServiceDuring)
+  optional string no_service_to = 3; 
+
+  // Rationale for setting the current status
+  optional string rationale = 4; 
+}
+
+message SetNodeOperationalStatus {
+  optional NodeOperationalStatusMetadata operational_status = 1; // Node's operational status to set
+  optional string worker_id = 2; // Storage/Distribution Worker ID
```

Comment:
Correct for storage nodes, but for distributor nodes we also need to know about the `worker_id`, since a bucket can be assigned to multiple operators?